### DISCUSSION TITLE
Everything is protected?

### DISCUSSION THREAD

**Discussion by Calgarybigfoot** [GriefPrevention/GriefPrevention#783] [Q&A] [answered]
I love the features and it works well, however...

I would like to set it so people can harvest and mine in areas outside of their claimed areas. Right now users can only mine cut trees etc in their claimed zones. I would like them to be able to build, mine etc in unclaimed areas as well as their protected zones.

right now everything seems protected.

How do I fix this?

**Comment by RoboMWM** [comment, marked as answer]
see documentation, you probably have that world set to Creative claims. Change it to survival.

**Comment by Calgarybigfoot** [comment]
i will give it a try. thank you
